ims/einstein/ceph.py

```diff
@@ -48,7 +48,7 @@ def __init_context(self):
 
     @trace
     def __init_cluster(self):
-        cluster = rados.Rados(rados_id=self.rid, conffile=self.r_conf)
+        cluster = rados.Rados(conffile=self.r_conf)
```

Comment:
This is the error I get if I don't make this change:

https://pastebin.com/kMsE4ij9